Dynatrace/dynatrace-operator
# Reduce RBAC permissions requested for operator/webhook

## Description

[K8S-11081](https://dt-rnd.atlassian.net/browse/K8S-11081)

Removes permissions that are requested that are not used.

Also removes some dead edgeconnect code

## How can this be tested?

Deploy the operator, deploy CRs, use as many features as possible, monitor the logs, and check there are no permission errors.
